src/Framework/AspNetCoreAnalyzers/src/Analyzers/RouteEmbeddedLanguage/FrameworkParametersCompletionProvider.cs

```diff
@@ -263,7 +263,7 @@ public override async Task ProvideCompletionsAsync(CompletionContext context)
 
             if (change.NewPosition != null)
             {
-                properties.Add(NewPositionKey, change.NewPosition.ToString());
+                properties.Add(NewPositionKey, change.NewPosition.Value.ToString(CultureInfo.InvariantCulture));
```

Comment:
Since we've already checked for null, it is marginally better to say

```suggestion
                properties.Add(NewPositionKey, change.NewPosition.GetValueOrDefault().ToString(CultureInfo.InvariantCulture));
```

So you don't pay for the null check again.

Reply:
I guess this was fixed in .NET 6. https://github.com/dotnet/runtime/issues/13699#issuecomment-902961514